### PR TITLE
Correct README

### PR DESCRIPTION
**Pull request by vincent-pli**
Since we have more than one `crd` definition (`Config`, `Addon`), so the original cmd will raise exception.

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ The following steps will install [Tekton Pipeline](https://github.com/tektoncd/p
     
 2. Apply Operator CRD
 
-    `kubectl apply -f deploy/crds/*_crd.yaml`
+    `kubectl apply -f deploy/crds/operator_v1alpha1_config_crd.yaml`
```
Comment (sthaha) [inline README.md]: isn't the old version better? i.e. apply all (`*`) crd.yaml files?

Comment (vincent-pli) [reply README.md]: Will raise exception if the number of files is more than one

Comment (sthaha) [reply README.md]: 🐼 learned something today 😀

**Pull request merged**